Automattic/wp-calypso
# Fix: business hours object order

#### Changes proposed in this Pull Request

The schema for business hours needs to be an array instead of an object, or else the days will become out of order. Props @lezama for reporting.

#### Testing instructions

* To reproduce the bug, visit one of your wordpress.com testing sites, and open the post editor in wp-admin
* Add a business hours block
* make a change to one of the hours, and observe that the days become out of order
* observe that the days of the week go out of order

To test:
- use this branch to build the blocks into your local jetpack site
- ensure the bug is gone
